llm_bioinformatics_research/src/UserProfile/UserProfile.js

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from 'react';
+import { Box, Typography, Button, IconButton, TextField, MenuItem, Select, FormControl, InputLabel, AppBar, Toolbar, Avatar } from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import EditIcon from '@mui/icons-material/Edit';
+import { useNavigate } from 'react-router-dom';
+import { ManageAccounts, Settings, Logout } from '@mui/icons-material';
+import './UserProfile.css';
+
+const UserProfile = () => {
+  const [section, setSection] = useState('profile'); // Default section is set to 'profile'
+  const [user, setUser] = useState({
+    image: '',
+    name: 'John Doe',
+    email: 'john.doe@example.com',
+    phone: '123-456-7890',
+    location: 'New York, USA',
+    theme: 'light',
+    language: 'en'
+  });
+  const navigate = useNavigate();
+
+  const handleInputChange = (e) => {
+    const { name, value } = e.target;
+    setUser({ ...user, [name]: value });
+  };
+
+  const handleImageChange = (e) => {
+    const file = e.target.files[0];
+    if (file) {
+      const reader = new FileReader();
+      reader.onloadend = () => {
+        setUser({ ...user, image: reader.result }); 
+      };
+      reader.readAsDataURL(file);
+    }
+  };
+
+  const handleLogout = () => {
+    navigate('/login'); // Placeholder for actual logout logic
+  };
+
+  return (
+    <Box className="userProfile-container">
+      <AppBar position="static" className="appBar">
+        <Toolbar>
+          <Typography variant="h4" className="title">
+            User Profile
+          </Typography>
+          <IconButton color="inherit" onClick={() => navigate('/home')}>
+            <CloseIcon />
+          </IconButton>
+        </Toolbar>
+      </AppBar>
+
+      <Box className="button-section">
+        <Button
+          onClick={() => setSection('profile')}
+          className={section === 'profile' ? 'selected-button' : 'default-button'}
+        >
+          <ManageAccounts /> My Profile
+        </Button>
+
+        <Button
+          onClick={() => setSection('settings')}
+          className={section === 'settings' ? 'selected-button' : 'default-button'} 
```

Comment:
When the buttons are selected, they don't remain highlighted to indicate that the user is on that section of the references page. For example, when I click on my profile, it isn't highlighted to indicate that I am on the my profile section. This might be due to conflicts between material UI and the css elements.